packages/frontend/src/features/videos/providers/Slide.tsx

```diff
@@ -67,33 +76,19 @@ class Slide extends React.Component<SlideProps, SlideState> {
       gsapMovies.length,
       margin
     );
-    let x = e.target.name === Directions.RIGHT ? move : -move;
-
-    if (videoPosition >= 0 && e.target.name === Directions.RIGHT) {
-      x = end;
-    } else if (videoPosition <= end && e.target.name === Directions.LEFT) {
-      x = -end;
-    } else if (
-      videoPosition === -lastMove &&
-      e.target.name === Directions.RIGHT &&
-      additionalVideosNum !== 0
-    ) {
-      x = lastMove;
-    } else if (
-      videoPosition === end + lastMove &&
-      e.target.name === Directions.LEFT &&
-      additionalVideosNum !== 0
-    ) {
-      x = -lastMove;
-    }
-    let time = Math.abs(x) === Math.abs(end) ? 0.8 : 0.6;
+    const hasAddidionalvideos  = additionalVideosNum !== 0 ? true : false
```

Comment:
wyrażenie 3 argumentowe nie potrzebne additionalVideosNum !== 0 to operacja która zwróci true/false